draft-ietf-rtcweb-jsep.xml

```diff
@@ -2824,6 +2824,35 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             with CN codecs and "usedtx=0").</t>
           </section>
         </section>
+        <section title="Direction Attribute in Answers"
+        anchor="sec.answer-dirattr">
+
+          <t><xref target="RFC3264"></xref> direction attributes
+          (defined in Section 6.1) in answers are chosen according to
+          the direction attribute in the remote offer and the direction
+          of the corresponding RtpTransceiver, by choosing the
+          intersection of the two directions in the following table:</t>
+
+          <figure>
+            <artwork>
+              <![CDATA[
+                     +---------------------------------------------------+
+                     |                  RtpTransceiver                   |
+                     +------------+------------+------------+------------+
+                     |  sendrecv  |  sendonly  |  recvonly  |  inactive  |
+    +---+------------+------------+------------+------------+------------+
+    |   |  sendrecv  |  sendrecv  |  sendonly  |  recvonly  |  inactive  |
+    | o +------------+------------+------------+------------+------------+
+    | f |  sendonly  |  recvonly  |  inactive  |  recvonly  |  inactive  |
+    | f +------------+------------+------------+------------+------------+
+    | e |  recvonly  |  sendonly  |  sendonly  |  inactive  |  inactive  |
+    | r +------------+------------+------------+------------+------------+
+    |   |  inactive  |  inactive  |  inactive  |  inactive  |  inactive  |
+    +---+------------+------------+------------+------------+------------+
```

Comment:
I opted for a hand-drawn table, since this lets us get away with a 5x5 table instead of a 3x17 flattened table (one row for each combination).

I considered just writing "choose the directional attribute that's the closest match to the RtpTransceiver's direction within the set of allowed values." But I thought maybe "closest match" would be too vague.